<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form4</class>
 <widget class="QWidget" name="Form4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Form4{
	background-color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label{
    text-align:right;

	font: 36pt &quot;华文楷体&quot;;
    font-size:35px;
    font-weight:bold;
	color: black;
    }</string>
   </property>
   <property name="text">
    <string>智慧火灾监控预警系统日志</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>891</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>881</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox{
    text-align:right;
    
	font: 15pt &quot;华文楷体&quot;;
    
    font-weight:bold;
	color: black;
    }</string>
   </property>
   <property name="title">
    <string>功能区</string>
   </property>
   <widget class="QPushButton" name="sel_all">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	/*灰色背景*/
	background:rgba(220,221,221,1);
	/*渐变白灰背景*/
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.03, y2:1, stop:0 rgba(253, 253, 253, 255), stop:1 rgba(238, 238, 238, 255));
	border:1px solid rgba(0, 0, 0, 0.23);
	border-radius:5px;
color: black;
font-family:'Microsoft YaHei';
}
/*悬停*/  
QPushButton:hover
{
    background-color:rgb(234,244,252);
	border:1px solid rgb(44,169,225);
}
/*按下*/  
QPushButton:pressed
{
	border:1px solid rgb(44,169,225);
    background-color:rgb(235,246,247);
    /*左内边距为2像素，让按下时字向右移动2像素*/  
    padding-left:2px;
    /*上内边距为2像素，让按下时字向下移动2像素*/  
    padding-top:2px;
}
</string>
    </property>
    <property name="text">
     <string>查询全部</string>
    </property>
   </widget>
   <widget class="QPushButton" name="return_bt">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	/*灰色背景*/
	background:rgba(220,221,221,1);
	/*渐变白灰背景*/
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.03, y2:1, stop:0 rgba(253, 253, 253, 255), stop:1 rgba(238, 238, 238, 255));
	border:1px solid rgba(0, 0, 0, 0.23);
	border-radius:5px;
color: black;
font-family:'Microsoft YaHei';
}
/*悬停*/  
QPushButton:hover
{
    background-color:rgb(234,244,252);
	border:1px solid rgb(44,169,225);
}
/*按下*/  
QPushButton:pressed
{
	border:1px solid rgb(44,169,225);
    background-color:rgb(235,246,247);
    /*左内边距为2像素，让按下时字向右移动2像素*/  
    padding-left:2px;
    /*上内边距为2像素，让按下时字向下移动2像素*/  
    padding-top:2px;

}
</string>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_bt">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	/*灰色背景*/
	background:rgba(220,221,221,1);
	/*渐变白灰背景*/
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.03, y2:1, stop:0 rgba(253, 253, 253, 255), stop:1 rgba(238, 238, 238, 255));
	border:1px solid rgba(0, 0, 0, 0.23);
	border-radius:5px;
color: black;
font-family:'Microsoft YaHei';
}
/*悬停*/  
QPushButton:hover
{
    background-color:rgb(234,244,252);
	border:1px solid rgb(44,169,225);
}
/*按下*/  
QPushButton:pressed
{
	border:1px solid rgb(44,169,225);
    background-color:rgb(235,246,247);
    /*左内边距为2像素，让按下时字向右移动2像素*/  
    padding-left:2px;
    /*上内边距为2像素，让按下时字向下移动2像素*/  
    padding-top:2px;
}
</string>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QPushButton" name="condit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	/*灰色背景*/
	background:rgba(220,221,221,1);
	/*渐变白灰背景*/
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.03, y2:1, stop:0 rgba(253, 253, 253, 255), stop:1 rgba(238, 238, 238, 255));
	border:1px solid rgba(0, 0, 0, 0.23);
	border-radius:5px;
color: black;
font-family:'Microsoft YaHei';
}
/*悬停*/  
QPushButton:hover
{
    background-color:rgb(234,244,252);
	border:1px solid rgb(44,169,225);
}
/*按下*/  
QPushButton:pressed
{
	border:1px solid rgb(44,169,225);
    background-color:rgb(235,246,247);
    /*左内边距为2像素，让按下时字向右移动2像素*/  
    padding-left:2px;
    /*上内边距为2像素，让按下时字向下移动2像素*/  
    padding-top:2px;
}
</string>
    </property>
    <property name="text">
     <string>条件查询</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>901</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*表格的一种美化方式*/
QHeaderView
{
    background:transparent;
}

QHeaderView::section
{
    font-size:18px;
    font-family:&quot;Microsoft YaHei&quot;;
    color:#FFFFFF;
    background:rgb(63, 63, 190);
    border:none;
    text-align:left;
    min-height:49px;
    max-height:49px;
    margin-left:0px;
    padding-left:0px;
}

QTableWidget
{
    background:#FFFFFF;
    border:none;

    font-size:20px;
    font-family:&quot;Microsoft YaHei&quot;;
    color:#666666;
}
QTableWidget::item
{
    border-bottom:1px solid #EEF1F7 ;
}

QTableWidget::item::selected
{
    color:black;
    background:#EFF4FF;
}


QScrollBar::handle:vertical
{
    background: rgba(255,255,255,20%);
    border: 0px solid grey;
    border-radius:3px;
    width: 8px;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical
{
    background:rgba(255,255,255,10%);
}


QScollBar::add-line:vertical, QScrollBar::sub-line:vertical
{
    background:transparent;
}
</string>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>地区</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>摄像头ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>具体位置</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>日志信息</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>日期</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>时间</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_2{
	
	font: 16pt &quot;楷体&quot;;
}</string>
   </property>
   <property name="text">
    <string>数据库IP</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#textBrowser{	
	font: 11pt &quot;Eras Demi ITC&quot;;

}</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Eras Demi ITC'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:10pt; font-weight:600;&quot;&gt;127.0.0.1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_3{
	
	font: 16pt &quot;楷体&quot;;
}</string>
   </property>
   <property name="text">
    <string>端口号</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#textBrowser{	
	font: 11pt &quot;Eras Demi ITC&quot;;

}</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;3306&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
